shaders/src/material_inputs.fs

```diff
@@ -57,6 +57,13 @@ struct MaterialInputs {
 #if defined(MATERIAL_HAS_POST_LIGHTING_COLOR)
     vec4  postLightingColor;
 #endif
+
+#if defined(MATERIAL_HAS_IOR)
+    float ior;
+    float absorption;
```

Comment:
Not sure we want to enable all properties when ior is set. We should discuss this.